src/main/java/pl/byczazagroda/trackexpensesappbackend/exception/GlobalExceptionHandlerController.java

```diff
@@ -8,6 +8,9 @@
 import pl.byczazagroda.trackexpensesappbackend.exception.AppException;
 import pl.byczazagroda.trackexpensesappbackend.exception.AppRuntimeException;
 
+/**
+ * GlobalExceptionHandlerController exception handler for all application.
```

Comment:
Suggested content:
" GlobalExceptionHandlerController exception handler for all application **exceptions**."

Reply:
Corrected.